src/main/java/com/mdabrow9/ticketbookingapp/api/v1/Model/MovieDTO.java

```diff
@@ -0,0 +1,21 @@
+package com.mdabrow9.ticketbookingapp.api.v1.Model;
+
+import lombok.Data;
+
+
+
+@Data
+public class MovieDTO implements Comparable<MovieDTO>
+{
+
+    private String title;
+
+    private String description;
+    private int durationTime;
+
+    @Override
+    public int compareTo(MovieDTO o)
+    {
+        return this.title.compareTo(o.title);
```

Comment:
Czy to zadziała:
1. z wielkimi i małymi literami
2. z polskimi znakami?
W jaki sposób moglibyśmy się o tym przekonać?